src/platforms/common/enriching-events/identify-user.mdx

```diff
@@ -66,7 +66,8 @@ The user segment, for apps that divide users in user segments.
 
 ### `ip_address`
 
-The user's IP address. If the user is unauthenticated, Sentry uses the IP address as a unique identifier for the user. Sentry will attempt to pull this from the HTTP request data (`request.env.REMOTE_ADDR` field in JSON), if available. That might require <PlatformIdentifier name="send-default-pii" /> set to `true` in the SDK options.
+The user's IP address. If the user is unauthenticated, Sentry uses the IP address as a unique identifier for the user.
+Serverside SDKs that instrument incoming requests will attempt to pull the IP address from the HTTP request data (`request.env.REMOTE_ADDR` field in JSON), if available. <PlatformSection notSupported={["javascript", "node"]}>That might require <PlatformIdentifier name="send-default-pii" /> set to `true` in the SDK options.</PlatformSection><PlatformSection supported={["javascript", "node"]}>This requires you to set the `include.ip` option to `true` in the [`RequestData`](https://docs.sentry.io/platforms/node/guides/connect/configuration/integrations/default-integrations/#requestdata) integration.</PlatformSection>
```

Comment:
> This requires you to set the `include.ip` option to `true` in the [`RequestData`]

Doesn't this only apply to node?

Reply:
Hmm - I'd rather have it be the correct source of truth, this split is something we should solve as an aside (cc @lizokm)

Reply:
Aight will update!

Reply:
Should I remove the entire paragraph for node though?

Reply:
It's still technically true for node - the ip address will just be of the server then though. Let's keep it, maybe we can come back to it later.

Reply:
Sorry, edited my previous comment. I meant the paragraph is technically not relevant for browser.

Reply:
Yes, I think that's a good idea.

Reply:
Discussed in person: too much headache in the include-exclude logic so we'll leave it in.